datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/index.md

```diff
@@ -5,66 +5,63 @@ keywords: Docker, UCP, troubleshoot
-are deployed on a node, depend on whether the node is a manager or worker.
-[Learn more about the UCP architecture](../../architecture.md)
-
-To check the state and logs of other UCP internal components, go to the
-**Containers** page, and appply the **System containers** filter.
-This can help validate that all UCP internal components are up and running.
-
-![](../../images/monitor-ucp-3.png){: .with-border}
+## Check the cluster status from the CLI
 
-It's normal for the `ucp-reconcile` to be stopped. This container only runs when
-the `ucp-agent` detects that a UCP internal component should be running but for
-some reason it's not. In this case the `ucp-agent` starts the `ucp-reconcile`
-service to start all UCP services that need to be running. Once that is done,
-the `ucp-agent` stops.
+You can also monitor the status of a UCP cluster using the Docker CLI client, by
+using a [client certificate
+bundle](../../user/access-ucp/cli-based-access.md). 
 
-## Check the cluster status from the CLI
+After that, you can view the status of all UCP nodes with the following command:
 
-You can also monitor the status of a UCP cluster using the Docker CLI client.
-There are two ways to do this, using a
-[client certificate bundle](../../user/access-ucp/cli-based-access.md), or logging into
-one of the manager nodes using ssh.
+```none
+$ docker node ls
+```
 
-Then you can use regular Docker CLI commands to check the status and logs
-of the [UCP internal services and containers](../../architecture.md).
+If any node is reported as `Down`, you may cross-reference whether that is an
+expected state by looking up the node's status message on the [full list of node
+states](./troubleshoot-node-messages.md). As a rule of thumb, if the status
+message starts with `[Pending]`, then the current state is transient and the
+node is expected to correct itself back into a healthy state eventually.
 
 ## Automated status checking
 
-You can use the `https://<ucp-url>/_ping` endpoint to perform automated
-monitoring tasks. When you access this endpoint, UCP validates that all its
-internal components are working, and returns the following HTTP error codes:
+You can use the `https://<manager-url>/_ping` endpoint to check the health of a
+single UCP manager node. When you access this endpoint, the UCP manager
+validates that all its internal components are working, and returns one of the
+following HTTP error codes:
 
 * 200, if all components are healthy
 * 500, if one or more components are not healthy
 
-If you're accessing this endpoint through a load balancer, you'll have no way to
-know which UCP manager node is not healthy. So make sure you make a request
-directly to each manager node.
+If an admin's client certificate is used as a TLS client certificate for the
+`_ping` endpoint, then a detailed error message will be returned if any
+components are unhealthy. 
+
+If you're accessing the `_ping` endpoint through a load balancer, you'll have no
```

Comment:
This is awesome! We tend to assume that this is obvious, but it's the kind of thing that is only obvious in hindsight. It really helps having this, specially when you're troubleshooting and starting to have tunnel vision.